sonic-xcvrd/tests/test_xcvrd.py

```diff
@@ -443,6 +443,27 @@ def test_CmisManagerTask_handle_port_change_event(self):
         task.on_port_update_event(port_change_event)
         assert len(task.port_dict) == 1
 
+    
+    @patch('xcvrd.xcvrd.XcvrTableHelper')
+    def test_CmisManagerTask_get_configured_freq(self, mock_table_helper):
```

Comment:
Can we add failure case? For instance, 1. when config_freq is out of allowed freq range. 2. when config_freq does not fall on 75GHz grid.

Reply:
This API tests get_configured_laser_freq() which obtains the configured value from the PORT table of CONFIG_DB. Invalid frequency is validated here https://github.com/Azure/sonic-utilities/pull/2197

Reply:
Am I looking at the correct place https://github.com/prgeor/sonic-utilities/blob/ba58d985b2483d24fb68821f11c6507dd08de567/scripts/portconfig#L337? seems like it's only validating the case when freq is <= 0?